com.unity.render-pipelines.universal/Runtime/Passes/DepthNormalOnlyPass.cs

```diff
@@ -67,13 +65,8 @@ public void Setup(RenderTextureDescriptor baseDescriptor, RenderTargetHandle dep
         public override void OnCameraSetup(CommandBuffer cmd, ref RenderingData renderingData)
         {
             if (this.allocateNormal)
-                cmd.GetTemporaryRT(normalHandle.id, normalDescriptor, FilterMode.Point);
-            if (this.allocateDepth)
-                cmd.GetTemporaryRT(depthHandle.id, depthDescriptor, FilterMode.Point);
-            ConfigureTarget(
-                new RenderTargetIdentifier(normalHandle.Identifier(), 0, CubemapFace.Unknown, -1),
-                new RenderTargetIdentifier(depthHandle.Identifier(), 0, CubemapFace.Unknown, -1)
-            );
+                cmd.GetTemporaryRT(Shader.PropertyToID(normal.name), normalDescriptor, FilterMode.Point);
```

Comment:
This is a screenspace camera texture, so I think it has to be converted to RTHandle otherwise there will be a clash of two dynamic scaling systems.

Reply:
What happened to `allocateDepth` and `depthRT` creation?